public/js/monitor.js

```diff
@@ -306,6 +306,67 @@ function addBentoObserver(){
     });
   });
 
+  document.querySelectorAll(".relay-sign-up-btn").forEach(btn => {
+    btn.addEventListener("click", async(e) => {
+      const relayEndpoint = new URL("/relay-waitlist", document.body.dataset.serverUrl);
+      const signUpCallout = document.querySelector(".relay-sign-up");
+
+      signUpCallout.classList.add("sending-email");
+      try {
+        await fetch(relayEndpoint, {
+          headers: {
+            "Content-Type": "application/json; charset=utf-8",
+          },
+          mode: "same-origin",
+          method: "POST",
+          body: JSON.stringify({"emailToAdd": "add-user-email"}),
+        })
+        .then(async(data) => {
+          setTimeout(()=> {
+            signUpCallout.classList.add("email-sent");
+            signUpCallout.classList.remove("sending-email");
+          }, 500);
+        });
+      } catch(e) {
+        // we need error messaging
+      }
+    });
+  });
+
+  const privateRelayCtas = document.querySelectorAll(".private-relay-cta");
+
+  if (privateRelayCtas.length > 0) {
```

Comment:
Can we refactor this into helper `function` like `addPrivateRelayCtaObservers` ? And then add an early `return;` if there are none.

```js

function addPrivateRelayCtaObservers() {
  const privateRelayCtas = document.querySelectorAll(".private-relay-cta");
  if (privateRelayCtas.length <= 0) {
    return;
  }

  const availableIntersectionObserver = ("IntersectionObserver" in window);
  ...
}
```